### DISCUSSION TITLE
plugins on openSUSE?

### DISCUSSION THREAD

**Discussion by bobblkabb** [hyprwm/Hyprland#5545] [Q&A]
Is there anyone here who has plugins (hyprpm) successfully running on openSUSE Tumbleweed?
I can't get it to work with the plugins. I have installed the appropriate packages as recommended in the wiki, but I always get a message about missing headers.
I also don't get a reasonable output when I run 'hyprctl version' btw.
```
Hyprland, built from branch at commit dirty ().
Date: 
Tag: 
flags: (if any)
```

**Comment by vaxerski** [comment]
that'd be your problem. At least the hash is necessary. Probably bring it up with the package maintainers.

**Comment by bobblkabb** [comment]
You're right: [openSUSE Bug Report](https://bugzilla.opensuse.org/show_bug.cgi?id=1218422)
